.github/workflows/frontend-integration-test.yml

```diff
@@ -94,7 +94,7 @@ jobs:
 
       - name: Upload integrate tests reports
         uses: actions/upload-artifact@v3
-        if: ${{ failure() && steps.integrationTest.outcome == 'failure' }}
+        if: ${{ (failure() && steps.integrationTest.outcome == 'failure') || contains(github.event.pull_request.labels.*.name, 'container log') }}
```

Comment:
It's not just the container log, the tag name is not proper. Use "log uploaded"? Also please change the related docs about integration test log related part.

Reply:
I have changed it to `upload log`.